ccd-definition/AuthorisationCaseField/la-solicitor.json

```diff
@@ -539,5 +539,26 @@
     "CaseFieldID": "cmoEventId",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "cmoDraftInReviewHeading",
+    "UserRole": "caseworker-publiclaw-solicitor",
```

Comment:
Could the permissions be appliedo n the Case role? [LASOLICITOR] I think it doesn't matter in this instance, but ideally we should use the case roles I think

Reply:
done